R/github-labels.R

```diff
@@ -1,22 +1,41 @@
-#' Create standard github labels for labelling issues.
+#' Manage GitHub issue labels.
 #'
-#' This function creates new labels and changes colours (if needed).
-#' It does not generally remove labels, unless you explicit ask to
-#' remove GitHub's defaults.
+#' @description `use_github_labels()` creates new labels and/or changes label
+#' colours. It does not generally remove labels, unless you explicitly ask to
```

Comment:
unless explicitly requested with `remove = TRUE`?

Reply:
Reworded to be mention `delete_default = TRUE`